src/session-pool.ts

```diff
@@ -405,19 +372,6 @@ export class SessionPool extends EventEmitter implements SessionPoolInterface {
     return this.size >= this.options.max!;
   }
 
-  /**
-   * Total number of read sessions.
-   * @type {number}
-   */
-  get reads(): number {
```

Comment:
Although these functions are not part of any interfaces, they could be used by users to gather statistics for their session pool usage, which means that also this would be a breaking change.

Reply:
interesting, I didn't consider this. Thanks for letting me know, reverted back.